__tests__/app.test.js

```diff
@@ -109,4 +109,39 @@ test('status: 404, responds with the correct status', () => {
   .get('/api/articlessss')
   .expect(404)
 })
-})
\ No newline at end of file
+})
+
+
+describe('GET /api/articles/:article_id/comments', () => 
+{
+  test('status:200, responds wth correct status', () => {
+    return request(app)
+      .get('/api/articles/1/comments')
+      .expect(200);    
+})
+test('check endpoint responds with the correct data', () => {
+  return request(app)
+  .get('/api/articles/1/comments').then(({body}) => 
+  {
+    console.log(body)
+    expect(typeof body).toEqual('object')
+    expect(body.data[0].comment_id).toBe(5)
+    expect(body.data[0].body).toBe('I hate streaming noses')
+    expect(body.data[0].article_id).toBe(1)
+    expect(body.data[0].author).toBe('icellusedkars')
+    expect(body.data[0].votes).toBe(0)
+    expect(body.data[0].created_at).toBe('2020-11-03T21:00:00.000Z')
```

Comment:
I don't think this is a firm enough test. This doesn't tell me "all the comments that came back belong to this article" it just tells me "the first comment looks like X" 

Please refactor this test to check that all the comments that came back belong to the article with ID 1. I want to know that nothing else has come through

Note: if you use a loop to do this then we want to make sure the right number of comments have come through (that an empty arr hasn't created a false positive)

Reply:
Done.